addons/website/models/website.py

```diff
@@ -527,7 +527,7 @@ def enumerate_pages(self, query_string=None):
                       of the same.
             :rtype: list({name: str, url: str})
         """
-        request.context = dict(request.context, **self.env.context)
+        # request.context = dict(request.context, **self.env.context)
```

Comment:
it was from same broken fix https://github.com/odoo/odoo/commit/a98d60cb11ceee31a6bd9d3734d0e4b565e5d677